packages/web3torrent/src/library/types.ts

```diff
@@ -78,6 +79,17 @@ export type PaidStreamingWire = Omit<Wire, 'requests'> &
     _onPiece(index: number, offset: number, buffer: Buffer): void;
   };
 
+export const isPaidStreamingWire = (obj: any): obj is PaidStreamingWire =>
+  typeof obj === 'object' && 'paidStreamingExtension' in obj;
+
+export type SerializedPaidStreamingWire = {
+  peerId: string;
+  amChoking: boolean;
```

Comment:
As an FYI, w3t never manually chokes or unchokes peers. Only `webtorrent` might do that under the hood.